frontend/class-frontend.php

```diff
@@ -849,6 +851,16 @@ public function robots_for_single_post( $robots, $post_id = 0 ) {
 		return $robots;
 	}
 
+
+	/**
+	 * Checks whether the user has written a meta-description. If written,  makes sure meta robots content is noodp.
+	 */
+	public function add_robot_content_noods() {
+		if ( ! ( WPSEO_Meta::get_value( 'metadesc' ) === '' ) && $this->options['noodp'] == false ) {
```

Comment:
Can you do a strict comparison, like:

`if ( ! ( WPSEO_Meta::get_value( 'metadesc' ) === '' ) && $this->options['noodp'] === false ) {`